src/libraries/Common/src/Interop/Windows/WinSock/Interop.TransmitFile.cs

```diff
@@ -14,7 +14,7 @@ internal static partial class Mswsock
         [DllImport(Interop.Libraries.Mswsock, SetLastError = true)]
         internal static extern unsafe bool TransmitFile(
             SafeHandle socket,
-            SafeHandle fileHandle,
+            SafeHandle? fileHandle,
```

Comment:
In what situation do we pass null here?

Reply:
In the case where `BeginSendFile(string? fileName, byte[]? preBuffer, byte[]? postBuffer` is called with a null or empty `fileName`.